samples/nextjs/src/lib/with-personalization-and-tracking.tsx

```diff
@@ -0,0 +1,60 @@
+import {
```

Comment:
Should `withCurrentPageTracking`, `withNotFoundPageTracking`, `withPersonalizationAndTracking` be moved to the `sitecore-jss-nextjs`? Not sure that this code should be here

Reply:
These HOCs require tracking service, therefore, require dependency on the tracking package. Currently, `sitecore-jss-nextjs` does not depend on tracking. We do not consider it acceptable either to add a new dependency or to introduce the new `sitecore-jss-nextjs-tracking` package. Therefore, we are out of options here.